### PR TITLE
[GBI-648] - On focus clear the content and show the place holder 

### PR DESCRIPTION
**Pull request by manfioLP**
https://rsklabs.atlassian.net/browse/GBI-648

https://user-images.githubusercontent.com/19415907/171922888-d827ac9d-d7e9-43d1-8311-6df1bc695b5d.mov

### CODE REVIEW

**Review by coolwather — COMMENTED** [review]
Nice work.
Just about the text I think could be "amount to swap" and in the other field "amount to receive".
For me was kind of weird to have placeholder in only one of the fields.

**Comment by manfioLP** [comment]
> Nice work. Just about the text I think could be "amount to swap" and in the other field "amount to receive". For me was kind of weird to have placeholder in only one of the fields.

agreed. now that you mentioned it started bugging me as well 😅
 changed it.

File: src/components/swap/SwapBox.vue [hunk 1]
```diff
@@ -43,7 +44,7 @@
           <span class="text-left ml-4">receive</span>
           <div class="currency-input">
             <div class="input-holder">
-              <input placeholder="0.0" v-model="swapTo.value" />
+              <input placeholder="amount to receive" v-model="swapTo.value" />
```
Comment (aguschirico) [inline src/components/swap/SwapBox.vue:47]: At a first sight looks good, but please rebase from main so I can test these changes integrated to the latest ones being merged into main.

**Review by aguschirico — CHANGES_REQUESTED** [review]
Looks good but please rebase master so we can test everything integrated.

**Pull request merged**